installation/index.html

```diff
@@ -19,10 +19,28 @@
 		<a href="#devel-panel" class=mdl-tabs__tab>Development</a>
 	</div>
 	<div class="mdl-tabs__panel is-active" id=binary-panel>
+		<aside class="card-note mdl-card mdl-shadow--2dp">
+			<div class="mdl-card__title">
+				<i class="material-icons">warning</i>
+				<div class="mdl-layout-spacer"></div>
+				Important!
+				<div class="mdl-layout-spacer"></div>
+				<i class="material-icons">warning</i>
+			</div>
+			<div class="mdl-card__supporting-text">
+				On OS X, be sure to install the OpenSSL package as root, the other packages can be installed as the user.
+				<pre class="r mdl-shadow--2dp"><code>>
+				user$ sudo R
+				> install.packages(c('openssl')
+				> quit()</code></pre>
```

Comment:
please don’t add `quit()` and how to start an R interpreter. people know how to do that 😉